androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/surveydetail/SurveyQuestionContent.kt

```diff
@@ -0,0 +1,189 @@
+package co.nimblehq.avishek.phong.kmmic.android.ui.screen.surveydetail
+
+import androidx.compose.foundation.*
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.layout.Arrangement.End
+import androidx.compose.foundation.pager.HorizontalPager
+import androidx.compose.foundation.pager.rememberPagerState
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.scale
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Color.Companion.White
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import co.nimblehq.avishek.phong.kmmic.android.R
+import co.nimblehq.avishek.phong.kmmic.android.ui.common.*
+import co.nimblehq.avishek.phong.kmmic.android.ui.theme.ApplicationTheme
+import co.nimblehq.avishek.phong.kmmic.presentation.uimodel.DisplayType.DROPDOWN
+import co.nimblehq.avishek.phong.kmmic.presentation.uimodel.SurveyQuestionUiModel
+import coil.compose.AsyncImage
+import kotlinx.coroutines.launch
+
+private const val TopGradientAlpha: Float = 0.01f
+private const val BottomGradientAlpha: Float = 0.6f
+private const val ImageScale = 1.5f
+
+@OptIn(ExperimentalFoundationApi::class)
+@Composable
+fun SurveyQuestionContent(
+    surveyQuestionUiModels: List<SurveyQuestionUiModel>,
+    onCloseClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    val pagerState = rememberPagerState()
+    val scope = rememberCoroutineScope()
+
+    Box(modifier = modifier) {
+        Box(Modifier.fillMaxSize()) {
```

Comment:
Redundant `Box`. Please move the content to the first `Box`. 🙏 
```suggestion
```

Reply:
Resolved in https://github.com/nimblehq/avishek-phong-kmm-ic/pull/120/commits/f686f7addab4416fb526524b008aed5a74ee2412